web-ui/src/main/resources/catalog/templates/editor/directory.html

```diff
@@ -1,107 +1,304 @@
 <div class="container" id="gn-directory-container">
-  <ul class="nav nav-pills">
-    <li class="disabled"><a data-translate=""
-                            href="javascript:void(0);">directoryManager</a></li>
-    <li data-ng-repeat="type in mdTypes"
-        data-ng-class="activeType === type ? 'active' : ''">
-      <a href="" data-ng-click="selectType(type)">
-        <i class="fa {{getTypeIcon(type)}}"/>&nbsp;{{type | translate}}</a></li>
-  </ul>
-  <br/>
-  <div class="row">
-    <div class="col-sm-4">
-      <div class="panel panel-default">
-        <div class="panel-heading">
-          <i class="fa fa-list-ul"/>
-          <span data-translate="">directoryEntries</span>
 
-          <div class="pull-right">
-            <a href="" data-ng-click="startImportEntry()">
-              <i class="fa fa-plus"/>
-            </a>
-          </div>
+  <!-- BROWSE MODE: editor is closed & entries are browsed by categories -->
+
+  <div class="row" ng-show="currentEditorAction == ''">
+    <div class="col-md-12">
+      <h1 translate>directoryManager</h1>
+      <p class="text-muted" translate>
+        directoryManagerSubtitle
+      </p>
+    </div>
+  </div>
+
+  <div class="row" ng-show="currentEditorAction == ''">
+    <div class="col-md-12 text-right">
+      <h5 class="btn-group">
+        <button type="button" class="btn btn-success"
+            ng-click="startImporting(false)">
+          <i class="fa fa-plus"/>&nbsp;
+          <span translate>addNewDirectoryEntry</span>
+        </button>
+        <button type="button" class="btn btn-success dropdown-toggle"
+          data-toggle="dropdown">
+          <i class="caret"/>
+        </button>
+        <ul class="dropdown-menu pull-right" role="menu">
+          <li class="dropdown-header text-left" translate
+            ng-repeat-start="(typeName, templates) in templates | groupBy: 'root'">
```

Comment:
This seems to be a problem with the backend; the query used for the dropdown is:
`qi?_content_type=json&template=t&fast=index&resultType=subtemplates&buildSummary=false`
and it returns literally every MD index.

Same for `q?_content_type=json&_isTemplate=t&fast=index` which returns everything, whereas `q?_content_type=json&_isTemplate=s&fast=index` returns only subtemplates.

Or did I understand something wrong?